src/domain/products/product-form/form/mappers.ts

```diff
@@ -95,10 +107,19 @@ export const formValuesToCreateProductMapper = (values, viewType) => {
     hs_code: values.hs_code,
     is_giftcard: false,
     discountable: values.discountable,
+    ...scData,
```

Comment:
suggestion: instead of returning the object, we can store the object in a const

Reply:
suggestion: after the const assigned we can move line 46 after it and assign the sales channels directly to the const object if the flag is enabled

Reply:
Did that initially but went for this solution instead because it's less obtrusive to eventually remove when the feature is globally enabled

Reply:
🤘